images/gitness/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. gitness)."
+  default     = ["gitness", "git", "busybox"]
```

Comment:
I'd make this `gitness` and remove it from the yaml, so we can control other versions we add to this image from outside the module.

Reply:
Fixed!